konfera/settings.py

```diff
@@ -70,3 +70,14 @@
 How long should we wait to notify about missing payment
 """
 UNPAID_ORDER_NOTIFICATION_REPEAT_DELAY = getattr(settings, 'UNPAID_ORDER_NOTIFICATION_REPEAT_DELAY', 5)
+
+"""
+Absolute url base with protocol
```

Comment:
I will add it. I thought about testing it in code just for sure, but it is probably cheaper to get error soon and have it fixed than check every single time if it is in required format. Am I right?